crux-bench/time-series-bench/bin/download_dataset.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/env bash
+wget https://timescaledata.blob.core.windows.net/datasets/devices_small.tar.gz
```

Comment:
can do this in fewer lines with curl piped to tar, and save yourself the cleanup